.github/workflows/destroy-preview-environment.yml

```
@@ -0,0 +1,78 @@
+
+jobs:
+  deploy-dev-pr-environment:
+    if: contains(github.event.pull_request.labels.*.name, 'deploy-pr')
+    runs-on: ubuntu-latest
+    outputs:
+      env_name: ${{ steps.env-name.outputs.PR_ENV_NAME }}
+    steps:
+      - name: Clean Ref
+        id: clean-ref
+        shell: bash
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:3:34: Double quote to prevent globbing and word splitting

(shellcheck)